### PR TITLE
Added exchange rate downloading from the National Bank of Hungary

### PR DESCRIPTION
**Pull request by nagyv**

### CODE REVIEW

File: currency_rate_update/services/update_service_HU_MNB.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2009 CamptoCamp. All rights reserved.
+#    @author Nicolas Bessi
```
Comment (StefanRijnhart) [inline currency_rate_update/services/update_service_HU_MNB.py]: Your name here as well?

File: currency_rate_update/services/update_service_HU_MNB.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+#
+#    Abstract class to fetch rates from Yahoo Financial
```
Comment (StefanRijnhart) [inline currency_rate_update/services/update_service_HU_MNB.py:7]: Update comment

**Comment by StefanRijnhart** [comment]
Could you rebase and check travis result afterwards?

**Comment by pedrobaeza** [comment]
As there's no answer, I close. Feel free to reopen in case there's interest.

**Pull request closed without merging**